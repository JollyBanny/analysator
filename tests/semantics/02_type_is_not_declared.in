var a: t;

type t = array [1..2] of array [1..2] of integer;

begin

end.