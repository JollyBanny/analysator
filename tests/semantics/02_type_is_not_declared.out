[Line: 1; Column: 8]	type 't' is not declared
